map/mobile-map-search-and-route/README.md

```diff
@@ -1,21 +1,30 @@
-# Mobile Map Search And Route
+# Mobile map (search and route)
 
-Use locators and networks saved in an offline map.
+Display maps and use locators to enable search and routing offline using a Mobile Map Package.
 
-![](MobileMapSearchAndRoute.png)
+![Image of mobile map search and route](MobileMapSearchAndRoute.png)
+
+## Use case
+
+Mobile map packages make it easy to transmit and store the necessary components for an offline map experience including: transportation networks (for routing/navigation), locators (address search, forward and reverse geocoding), and maps. 
+
+A field worker might download a mobile map package to support their operations while working offline.
 
 ## How to use the sample
 
-Click the "Open mobile map package" button to bring up a file choosing dialog. Browse to and select a .mmpk file. When chosen, the maps inside the mobile map package will be displayed in a list view. If the mobile map package has a locator task, the list items will have a pin icon. If the map contains transportation networks, it will have a navigation icon. Click on a map list item to open it. Click on the map to reverse geocode the clicked locations's address if a locator task is available. If transportation networks are available, a route will be calculated between geocode locations.
+Click the "Open mobile map package" button to bring up a file choosing dialog. Browse to and select a .mmpk file. When chosen, the maps inside the mobile map package will be displayed in a list view.
+ Click on a map in the list to open it. 
 
-## How it works
+If the mobile map package has a locator task, the list items will have a pin icon. Click on the map to reverse geocode the clicked locations's address if a locator task is available. 
+
+If the map contains transportation networks, it will have a navigation icon. Click on a map list item to open it. If transportation networks are available, a route will be calculated between geocode locations.
```

Comment:
```suggestion
If the map contains transportation networks, it will have a navigation icon. Click on the map to add locations.  If transportation networks are available, a route will be calculated between  locations.
```